unime/src/routes/prompt/share-credentials/+page.svelte

```diff
@@ -69,9 +70,9 @@
   </div>
 
   <!-- Controls -->
-  <div class="sticky bottom-0 left-0 flex flex-col space-y-[10px] rounded-t-2xl bg-white p-6 dark:bg-dark">
+  <div class="dark:bg-dark sticky bottom-0 left-0 flex flex-col space-y-[10px] rounded-t-2xl bg-white p-6">
     <Button
-      label="Approve request"
+      label={$LL.ACCEPT()}
```

Comment:
Should we make a distinction between "Approve" and "Accept"? (It used to be "Approve request" and we're now changing it to "Accept", does that change the meaning?)